file_store/src/error.rs

```diff
@@ -41,6 +41,8 @@ pub enum Error {
     //Not recommended for internal use!
     #[error("external error")]
     ExternalError(#[from] Box<dyn std::error::Error + Send + Sync>),
+    #[error("general error")]
+    Anyhow(#[from] anyhow::Error),
```

Comment:
This doesn't look to be used anywhere

Reply:
good.. because that's terribel

Reply:
addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393